frontend/src/components/DivisionInputModal.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import Modal from './Modal';
+import { Controller, useForm } from 'react-hook-form';
+import Button from './Button';
+import * as yup from 'yup';
+import { NAME_REGEX } from '../common/utils/utils';
+import i18n from '../common/config/i18n';
+import { yupResolver } from '@hookform/resolvers/yup';
+import Input from './Input';
+
+interface DivisionInputModalProps {
+  title: string;
+  onClose: () => void;
+  onSubmit: (division: DivisionInputTypes) => void;
+  defaultValue?: string;
+}
+
+type DivisionInputTypes = {
```

Comment:
I think we should call this DivisionFormTypes, as event though it defines one input we are still using it do describe the form type.